### PR TITLE
Fix unsafe crash

### PR DESCRIPTION
**Pull request by achille-roussel**
Follow up of https://github.com/segmentio/events/pull/22 

Fixes the crash (mostly just disabling the unsafe optimization). I don't fully grasp why the GC complained here, I think it has to do with the fact that some heap allocated objects ended up with pointers to stack allocated values.

### CODE REVIEW

File: logger.go [hunk 1]
```diff
@@ -128,6 +128,10 @@ func (l *Logger) log(depth int, debug bool, format string, args ...interface{})
 
 	h.HandleEvent(&s.e)
 
+	for i := range s.e.Args {
```
Comment (f2prateek) [inline logger.go]: I'm not sure I understand what lines 131-133 are doing. Why is this required?

Comment (achille-roussel) [inline logger.go]: I should comment, thanks for pointing it out. This is to ensure that the backing array doesn't hold pointers before it is placed back into the pool. 
I'll add a comment.

**Comment by achille-roussel** [comment]
Is this good to go?

**Comment by achille-roussel** [comment]
Thanks for the review!
